integration_tests/router_support.go

```diff
@@ -34,6 +34,7 @@ func reloadRoutes(optionalPort ...int) {
 	resp, err := http.Post(fmt.Sprintf("http://127.0.0.1:%d/reload", port), "", nil)
 	Expect(err).NotTo(HaveOccurred())
 	Expect(resp.StatusCode).To(Equal(202))
+	resp.Body.Close()
```

Comment:
good catch!

Reply:
ah I can't take the credit, there's [a linter for this](https://github.com/golangci/golangci-lint/blob/master/pkg/golinters/bodyclose.go) :D